test-data/unit/semanal-basic.test

```diff
@@ -47,6 +47,14 @@ MypyFile:1(
       NameExpr(f [__main__.f])
       Args())))
 
+[case testAssignEmptyTuple]
```

Comment:
Not sure we need this test; these tests that print out internal data structures are harder to maintain and harder to map to user-visible behavior.